src/Message/PurchaseRequest.php

```diff
@@ -172,7 +172,7 @@ public function getData()
     public function getAmount()
     {
         if ($this->getParameter('multiply')) {
-            return (float)parent::getAmount() * 100;
+            return strval((float)parent::getAmount() * 100);
         }
 
         return (float)parent::getAmount();
```

Comment:
@libertogimenez  Not sure what is the issue with 7.3, but if the change in 175 solves the problem, shouldn't it be applied also here?

Reply:
Hi. Thank you for the quick response!!! We working only with the multiply "mode". We detect the problem after multiply by 100. In any case, I think strval() it works too in line 178.

Reply:
Ok, could you please also adapt line 178 in this PR so it can be merged?
Unfortunatelty It's been a long time since I don't have time to take care of this project and I am not even versioning it properly :(

Thanks for your update.

Reply:
Done! Thank you very much!!